File: task-03/Javascript/code3..js
```javascript
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const printDiamond = (n) => {
  for (let i = 0; i < n; i++) {
    console.log(' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1));
  }
  for (let i = n - 2; i >= 0; i--) {
    console.log(' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1));
  }
};

rl.question('Enter a number: ', (answer) => {
  const n = parseInt(answer);
  printDiamond(n);
  rl.close();
});
``

File: task-03/Rust/code3..rs

```rust
use std::io;

fn print_diamond(n: usize) {
    for i in 0..n {
        println!("{}{}", " ".repeat(n - i - 1), "*".repeat(2 * i + 1));
    }
    for i in (0..n-1).rev() {
        println!("{}{}", " ".repeat(n - i - 1), "*".repeat(2 * i + 1));
    }
}

fn main() {
    let mut input = String::new();
    println!("Enter a number: ");
    io::stdin().read_line(&mut input).unwrap();
    let n: usize = input.trim().parse().unwrap();
    print_diamond(n);
}
```
